docs/r2/image_summaries.ipynb

```diff
@@ -0,0 +1,639 @@
+        "\n",
+        "  # Calculate the confusion matrix.\n",
+        "  cm = sklearn.metrics.confusion_matrix(test_labels, test_pred)\n",
+        "  # Log the confusion matrix as an image summary.\n",
+        "  figure = plot_confusion_matrix(cm, class_names=class_names)\n",
+        "  cm_image = plot_to_image(figure)\n",
+        "\n",
+        "  # Log the confusion matrix as an image summary.\n",
+        "  with file_writer_cm.as_default():\n",
+        "    tf.summary.image(\"Confusion Matrix\", cm_image, step=epoch)\n",
+        "\n",
+        "# Define the per-epoch callback.\n",
+        "cm_callback = keras.callbacks.LambdaCallback(on_epoch_end=log_confusion_matrix)\n",
+        "\n",
+        "%tensorboard --logdir logs/image\n",
+        "\n",
+        "# Train the classifier.\n",
+        "model.fit(\n",
+        "    train_images,\n",
+        "    train_labels,\n",
+        "    epochs=5,\n",
+        "    verbose=0, # Suppress chatty output\n",
+        "    callbacks=[tensorboard_callback, cm_callback],\n",
+        "    validation_data=(test_images, test_labels),\n",
+        ")"
+      ],
+      "execution_count": 0,
+      "outputs": []
+    },
+    {
+      "metadata": {
+        "id": "o7PnxGf8Ur6F",
+        "colab_type": "text"
+      },
+      "cell_type": "markdown",
+      "source": [
+        "<img class=\"tfo-display-only-on-site\" src=\"https://github.com/tensorflow/tensorboard/blob/master/docs/r2/images/images_accuracy.png?raw=1\"/>\n",
+        "\n",
+        "<img class=\"tfo-display-only-on-site\" src=\"https://github.com/tensorflow/tensorboard/blob/master/docs/r2/images/images_cm.png?raw=1\"/>"
+      ]
+    },
+    {
+      "metadata": {
+        "id": "6URWgszz9Jut",
+        "colab_type": "text"
+      },
+      "cell_type": "markdown",
+      "source": [
+        "Notice that accuracy is climbing on both train and validation sets. That's a good sign. But how is the model performing on specific subsets of the data?\n",
+        "\n",
+        "Select the \"Images\" tab to visualize your logged confusion matrices.\n",
+        "Check \"Show actual image size\" at the top left to see the confusion matrix at full size.\n",
+        "\n",
+        "By default the dashboard shows the image summary for the last logged step or epoch. Use the slider to view earlier confusion matrices. Notice how the matrix changes significantly as training progresses, with darker squares coalescing along the diagonal, and the rest of the matrix tending toward 0 and white. This means that your classifier is improving as training progresses! Great work!\n",
+        "\n",
+        "The confusion matrix shows that this simple model has some problems. Despite the great progress, Shirts, T-Shirts, and Pullovers are getting confused with each other. The model needs more work.\n",
+        "\n",
+        "If you're interested, see how you can improve this model with a more complicated network."
```

Comment:
"more complicated" is not necessarily better :). Are there any guides we can point to for better results on FashionMNIST? Maybe even the keras guides with convolutional layers.

Reply:
Reworded and linked.